README.md

```diff
@@ -25,6 +25,11 @@ would match `3262.1` and `3262.1.1`, but not `3262.2`.
 * `force_regular`: *Optional.* Default `false`. By default, the resource will always download light stemcells for IaaSes that support light stemcells.
   If `force_regular` is `true`, the resource will ignore light stemcells and always download regular stemcells.
 
+* `force_light`: *Optional.* Default `false`. By default, the resource will always download light stemcells for IaaSes
+  that support light stemcells. If `force_light` is `true`, the resource will ignore regular stemcells and always
```

Comment:
what if both force_regular and force_light are set to true?

Reply:
If you set them both to true, the way the current code is structured you'll get regular stemcells. But I'd consider this undefined behavior, I can think of no use case that makes sense for setting both to true, so it would only be a mistake.

Reply:
my mistake here. I meant `force_regular` will take precedence based on line https://github.com/concourse/bosh-io-stemcell-resource/blob/3d36bd8c8653ba2b5e46984d6c22f7e79d1c171a/boshio/stemcells.go#L40

so in the doc of `force_regular`, i think it is better to explicity call it out.